pobvis/app/main.py

```diff
@@ -47,24 +47,53 @@ def pooling():
     update_status()
     return fetch_exps()
 
-def transform_exprs():
+def learn_transformation():
+    request_params = request.get_json()
+    exp_path = request_params.get('exp_path', '')
+    exp_folder = os.path.join(MEDIA, exp_path)
+    declare_statements = get_declare_statements(exp_folder)
+    body = {
+        'instance': exp_path,
+        'declareStatements': declare_statements
+    }
+    url = 'http://0.0.0.0:2000/api/v1/transformations/learntransformation'
```

Comment:
I would prefer url to be put in the settings.py file. Make it easier to change down the line